docs/request.md

```diff
@@ -27,7 +27,7 @@ Makes XHR (aka AJAX) requests, and returns a [promise](promise.md)
 m.request({
 	method: "PUT",
 	url: "/api/v1/users/:id",
-	data: {id: 1, name: "test"}
+	params: {id: 1, name: "test"}
```

Comment:
```suggestion
	params: {id: 1},
	body: {name: "test"}
```

Reply:
Why?

Reply:
The request method is `PUT`, so it needs a body. The current example will produce:

```
PUT /api/v1/users/1
```

with an empty body. 😨

Reply:
It works well enough for the examples, and I've done put-like operations using the query string only. Your app obviously will be more complicated, and this is purely just for show, nothing more.

Reply:
This is a fundamental change in this example. Previously, it would have made an `application/json` request without a query string.

If I make a PR to fix this, will you accept it?

Reply:
I'll consider it as long as you include this context, but keep in mind `id` has to be in `params` for it to be interpolated.

Reply:
https://github.com/MithrilJS/mithril.js/pull/2430